shell/src/main/java/alluxio/cli/fs/command/RmCommand.java

```diff
@@ -99,8 +107,10 @@ protected void runPlainPath(AlluxioURI path, CommandLine cl)
           + " please use \"rm -R/-r/--recursive <path>\"");
     }
     boolean isAlluxioOnly = cl.hasOption(REMOVE_ALLUXIO_ONLY.getLongOpt());
+    boolean isDeleteMountPoint = cl.hasOption(DELETE_MOUNT_POINT.getLongOpt());
```

Comment:
check the shorthand too?

Reply:
It is not necessary, because the resolveOption method can resolve the option by long or short option name.

Reference to 
```java
    private Option resolveOption(String opt)
    {
        opt = Util.stripLeadingHyphens(opt);
        for (Option option : options)
        {
            if (opt.equals(option.getOpt()))
            {
                return option;
            }

            if (opt.equals(option.getLongOpt()))
            {
                return option;
            }

        }
        return null;
    }

```